lbz/request.py

```diff
@@ -68,6 +50,43 @@ def _safe_json_loads(payload: str | bytes) -> Any:
         except ValueError as error:
             raise BadRequestError(f"Invalid payload.\nPayload body:\n {repr(payload)}") from error
 
+
+class Request(BaseRequest):
+    """Represents request from HTTP API Gateway."""
+
+    def __init__(
+        # pylint: disable=too-many-positional-arguments
+        self,
+        headers: CIMultiDict,
+        uri_params: dict,
+        method: str,
+        body: str | bytes | dict,
+        context: dict,
+        stage_vars: dict,
+        is_base64_encoded: bool,
+        query_params: dict | None = None,
+        user: User | None = None,
+    ):
+        self.query_params = MultiDict(query_params or {})
+        self.headers = headers
+        self.uri_params = uri_params
+        self.method = method
+        self.context = context
+        self.stage_vars = stage_vars
+        self.user = user
+        super().__init__(body=body, is_base64_encoded=is_base64_encoded)
+
+    def __repr__(self) -> str:
+        return f"<Request {self.method} >"
+
+    def to_dict(self) -> dict:
```

Comment:
As we agreed a while ago, all the properties should be implemented before methods like this (original order).